### PR TITLE
Add run scripts

### PR DESCRIPTION
**Pull request by DanielMwendwa**

### CODE REVIEW

File: run_scripts/1_coda_get.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+fi
+
+AUTH=$1
+CODA_V2_ROOT=$2
+DATA_ROOT=$3
+
+./checkout_coda_v2.sh "$CODA_V2_ROOT"
+
+DATASETS=(
+    "KE_VAX_vaccination_thoughts"
```
Comment (IsaackMwenda) [inline run_scripts/1_coda_get.sh]: we need to replace this with the research question datasets for this project.

File: run_scripts/1_coda_get.sh [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    "KE_VAX_other_messages"
+
+    "KE_Engagement_poll_chatting_apps"
+    "KE_Engagement_poll_access_to_internet"
+    "KE_Engagement_poll_facebook_usage"
+    "KE_Engagement_poll_radio_stations"
+    "KE_Engagement_poll_other_messages"
+
+    "Kenya_Pool_age"
```
Comment (IsaackMwenda) [inline run_scripts/1_coda_get.sh:23]: And only keep this.

File: run_scripts/4_coda_add.sh [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    exit
+fi
+
+AUTH=$1
+CODA_V2_ROOT=$2
+DATA_ROOT=$3
+
+./checkout_coda_v2.sh "$CODA_V2_ROOT"
+
+DATASETS=(
```
Comment (IsaackMwenda) [inline run_scripts/4_coda_add.sh:17]: ditto

**Review by IsaackMwenda — APPROVED** [review]

File: run_scripts/1_coda_get.sh (hunk 1 above)
Comment (DanielMwendwa) [reply run_scripts/1_coda_get.sh]: Thanks, updated.

File: run_scripts/4_coda_add.sh (hunk 3 above)
Comment (DanielMwendwa) [reply run_scripts/4_coda_add.sh:17]: Thanks, updated.

**Pull request merged**